pkg/server/server.go

```diff
@@ -59,6 +66,10 @@ func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		})
 	}
 
+	if eventType == string(github.PullRequest) || eventType == string(github.IssueComment) {
```

Comment:
shouldn't that be already filtered via plugins configuration in `plugins.yaml` file?